prost-build/src/code_generator.rs

```diff
@@ -1162,15 +1191,22 @@ fn build_enum_value_mappings<'a>(
 
         if let Some(old_v) = generated_names.insert(generated_variant_name.to_owned(), value.name())
         {
-            panic!("Generated enum variant names overlap: `{}` variant name to be used both by `{}` and `{}` ProtoBuf enum values",
-                generated_variant_name, old_v, value.name());
+            // if alias ends up being a duplicate, we don't need it, and can skip it.
+            // TODO: check if enum values are actually the same
```

Comment:
I think this TODO needs to be solved

Reply:
I feel like duplicates should always be disallowed. Even if they are an alias, then overlap is not disirable.